data/sheragi/archaeology missions.txt

```diff
@@ -635,16 +635,16 @@ mission "Sheragi Archaeology: The Box 2"
+			`	You take the chance to catch up as he leads you into the ancient city's ruins, talking about the "minor" discoveries he has made so far, though he insists on keeping the reason he brought you here a surprise. He leads you to one of the Sheragi buildings that have been uncovered, skipping a long queue lined before the entrance to find that the room has been filled with various artifacts in display cases.`
+			`	It's strange to see so many technological devices as old as some of the prehistoric tools from ancient Earth, most of them so deteriorated you wouldn't even be able to imagine what they were used for if it wasn't for the descriptions next to their display cases. One of the most intact relics is a small metal statue of an individual Sheragi in flight like the dragons of Skymoot, possibly a trophy for some event, or perhaps a mere sculpture of a popular character of their time. It's impossible to know now, but one thing is certain: it's the most faithful representation available of what they looked like. The figure matches almost perfectly the ones you saw in your dream, back in Ildaria. As you turn your head to look at how tall the doors of this house are, then back at the statue again, you realize how big they must have been. It's hard to imagine the size their spaceships must have been to even hope to fit a creature this large.`
 			choice
 				`	"This is an impressive collection."`
 			`	"It is! But it's not even the main attraction for you today. Follow me."`
 			`	After the tour is over, he leads you to another building in an area that is off-limits to the general public.`
 				goto offlimit
 			label hurry
 			`	"Straight to the point? Fair enough, I'm eager as well, but do come visit the museum some time in the future. There are so many interesting objects."`
-			`	You take the chance to catch up as he leads you into the ancient city's ruins, talking about the "minor" discoveries he made so far, but he quickly gets to the reason you're here: an artifact found buried even further underground. He leads you into one of the buildings, in an area that is off-limits to the public.`
+			`	You take the chance to catch up as he leads you into the ancient city's ruins, talking about the "minor" discoveries he made so far, such as statues of Sheragi that depict what they once looked like. "We would perhaps never know their appearance had we not found something like it. But these small discoveries are not the main attraction for you today," Albert says with a wide smile. He leads you into one of the uncovered Sheragi buildings, in an area that is off-limits to the public.`
```

Comment:
Similar to the change above to remove a duplicate mention of "museum," this section had a duplicate mention of going "further underground." Replaced that with a mention of one of the "minor discoveries" that Albert made so that it can be compared to the more major discovery of the black box.